app/reducers/events.js

```diff
@@ -406,7 +406,18 @@ export const selectAllRegistrationsForEvent = createSelector(
         const user = registration.user.id
           ? registration.user
           : usersById[registration.user];
-        return transformRegistration({ ...registration, user });
+        return transformRegistration({
+          ...registration,
+          user,
+          createdBy:
+            registration.createdBy !== null
+              ? usersById[registration.createdBy]
```

Comment:
Since the serializer returns only the `pk`, I guess`usersById[registration.createdBy]` and `usersById[registration.updatedBy]` can return `undefined`, or?

Reply:
I believe you are correct, I have added an `undefined` check to caa5fb4c3b8034028756d3e3fa421ebb9d2500e1.